### PR TITLE
Standalone repo config, yarn, circleci integration, shipit integration

### PR DESCRIPTION
**Pull request by GoodForOneFare**
See title.

### CODE REVIEW

File: scripts/deploy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var exec = require('child_process').exec;
```
Comment (lemonmade) [inline scripts/deploy.js]: Simplify deploy script :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
+.DS_STORE
+npm-debug.log*
+
```
Comment (lemonmade) [inline .gitignore:4]: Remove

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+node_modules
+.DS_STORE
+npm-debug.log*
```
Comment (lemonmade) [inline .gitignore]: I usually do `*.log`

**Review by lemonmade — CHANGES_REQUESTED** [review]
Same comment as previous PR re: deploy script, then looks good 👍

**Comment by GoodForOneFare** [comment]
Cleaned up issues common to these PRs & removed legacy `build`, `build:lib`, `preversion` scripts.  I think this is gtg.

**Review by lemonmade — APPROVED** [review]

**Pull request merged**